File: process/OFM/tidy_ofm_apr_post.py
# This script will tidy downloaded post-censal OFM April 1 data
import os

exec(open('functions_tidy.py').read())

data_dir = '../../data'

filename = 'ofm_april1_population_final.xlsx'
wa = pd.read_excel(os.path.join(data_dir, filename), skiprows = 4)
df_post = tidy_ofm_apr(wa, 'Population')

File: process/OFM/tidy_ofm_apr_post.R
# This script will tidy downloaded post-censal OFM April 1 data

source(here::here('process', 'OFM', 'config.R'))
source(here::here('process', 'OFM', 'functions_tidy.R'))

data_dir <- 'data'
filename <- filename_post
wa <- read.xlsx(here(data_dir, filename), startRow = 5)
df_post <-  tidy_ofm_apr(wa, 'Population')

rm(wa)